src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -789,6 +789,7 @@ def set_up_service_binds(self):
                                                                                             self.get_argument_service_bindings(),
                                                                                             self.get_argument_resource_group_name(),
                                                                                             self.get_argument_name(),
+                                                                                            self.containerapp_def["properties"]["environmentId"],
```

Comment:
recommend to use safe_get

` safe_get(self.containerapp_def, "properties", 'environmentId')`

Reply:
done